File: Aula 06_08/exercicio_1.py
lista = []

for i in range(10):
  lista.append(int(input(f'Insira o {i + 1}º número: ')))

soma = sum(lista)
media = soma/len(lista)
qtd_acima_media = 0
qtd_abaixo_media = 0

for numero in lista:
  if numero > media:
    qtd_acima_media += 1
  elif numero < media:
    qtd_abaixo_media += 1

print(f'Média: {media}')
print(f'Acima da média: {qtd_acima_media}')
print(f'Abaixo da média: {qtd_abaixo_media}')

File: Aula 06-08/exercicio_1.c
#include <stdio.h>

int main() {
  int tamanhoLista = 3;
  int lista[tamanhoLista];
  int soma = 0, qtdAcimaMedia = 0, qtdAbaixoMedia = 0;

  for (int i = 0; i < tamanhoLista; i++) {
    printf("Insira o %d número: ", i + 1);
    scanf("%d", &lista[i]);
    soma += lista[i];
  }

  float media = (float) soma / tamanhoLista;

  for (int i = 0; i < tamanhoLista; i++) {
    int numero = lista[i];

    if (numero > media) {
      qtdAcimaMedia++;
    } else if(numero < media) {
      qtdAbaixoMedia++;
    }

    soma += numero;
  }

  printf("Média: %f\n", media);
  printf("Acima da média: %d\n", qtdAcimaMedia);
  printf("Abaixo da média: %d\n", qtdAbaixoMedia);

  return 0;
}